src/tablet/tablet_impl.cc

```diff
@@ -644,6 +639,14 @@ void TabletImpl::Put(RpcController* controller, const ::openmldb::api::PutReques
         replicator->AppendEntry(entry);
     } while (false);
 
+    ok = UpdateAggrs(request->tid(), request->pid(), request->value(),
+                     request->dimensions(), replicator->GetLogOffset());
```

Comment:
the offset obtained  by `replicator->GetLogOffset()` may be not referenced this recored. if you want to use the offset of current record, `log_index` in entry is recommend